### PR TITLE
Add solution rates aproximation to metrics

### PR DESCRIPTION
**Pull request by oxarbitrage**
Closes https://github.com/zcash/zcash/issues/4571

### CODE REVIEW

File: src/metrics.cpp [hunk 1]
```diff
@@ -302,8 +307,8 @@ int printStats(bool isScreen, bool mining)
             if (netheight <= 0)
                 netheight = 1;
             int downloadPercent = height * 100 / netheight;
-            std::cout << "     " << _("Downloading blocks") << " | " << height << " (" << nHeaders << " " << _("headers") << ") / ~" << netheight << " (" << downloadPercent << "%)" << std::endl;
 
+            std::cout << "     " << _("Downloading blocks") << " | " << height << " (" << nHeaders << " " << _("headers") << ") / ~" << netheight << " (" << downloadPercent << "%)" << std::endl;
```
Comment (str4d) [inline src/metrics.cpp:311]: This diff is confusing; I think the blank line was moved up one line? Would be nice to undo this change.

File: src/metrics.cpp [hunk 2]
```diff
@@ -342,9 +347,9 @@ int printStats(bool isScreen, bool mining)
     }
     std::cout << "           " << _("Next upgrade") << " | " << strUpgradeTime << std::endl;
     std::cout << "            " << _("Connections") << " | " << connections << std::endl;
-    std::cout << "  " << _("Network solution rate") << " | " << netsolps << " Sol/s" << std::endl;
+    std::cout << "  " << _("Network solution rate") << " | " << netsolps << " Sol/s " << strprintf("(~ 2^%.4f Sol/s)", netsolpsLog) << std::endl;
```
Comment (str4d) [inline src/metrics.cpp:350]: Let' just replace the base-10 representation with the log-2 one for the general view:
```suggestion
    std::cout << "  " << _("Network solution rate") << " | " << strprintf("~ 2^%.4f Sol/s", netsolpsLog) << std::endl;
```

**Comment by str4d** [comment]
@zkbot r+

**Comment by daira** [comment]
Reminder that @therealyingtong is the release manager and should be asked to approve non-essential PRs, this close to a release.
